### DISCUSSION TITLE
CSS Nesting

### DISCUSSION THREAD

**Discussion by zeroedin** [RedHat-UX/red-hat-design-system#1215] [General]
https://caniuse.com/?search=nesting

Should we start looking at adopting?  Pros?  Cons?

Can always postCSS back to flat syntax as a back fill?

Generally our syntax is pretty terse in ShadowDOM CSS.  That said I can see this cleaning up some LightDOM styles for things like `rh-navigation-secondary`

**Comment by bennypowers** [comment]
I'm big on nesting, and think we can use it today in eleventy without issue, but I think we should avoid adding postcss transforms to shadow DOM (except for things like minification)

When browsers ship (soon!) We can just add a stylint rule to auto-fix flat->nested

**Reply by zeroedin** [reply]
I agree, I don't think it's even necessary in our shadowDOM CSS as it stands.  No complex selectors really necessary.  

Definitely see it as a bonus for LightDOM though.
